distributed/cli/dask_worker.py

```diff
@@ -16,18 +16,22 @@
 
 logger = logging.getLogger('distributed.dask_worker')
 
-
 import signal
 
+
 def handle_signal(sig, frame):
     loop = IOLoop.instance()
     if loop._running:
         loop.add_callback(loop.stop)
     else:
         exit(1)
 
-signal.signal(signal.SIGINT, handle_signal)
-signal.signal(signal.SIGTERM, handle_signal)
+
+def install_signal_handlers():
+    # NOTE: We can't use the generic install_signal_handlers() function from
+    # distributed.cli.utils because we're handling the signal differently.
+    signal.signal(signal.SIGINT, handle_signal)
+    signal.signal(signal.SIGTERM, handle_signal)
```

Comment:
Sure, I'd be happy to define the handler function inline if that's clearer -- I don't think there's much in it, though.